### ISSUE TITLE
Features from Abstract class do not appear inside the HTML report

### ISSUE THREAD

**Issue by Auliyaa** [renatoathaydes/spock-reports#187]
This issue is a follow-up to the discussion from issue #182 .
Given the following code:

```groovy
// TestSpec.groovy

class TestSpec extends AbstractSpec {
  def setup() {
    println("setting-up")
  }
}

// AbstractSpec.groovy
import spock.lang.Specification

class AbstractSpec extends Specification {
  def "My test case"() {
    when:
    def a = 1+1
    then:
    a == 2
  }
}
```

When generating the spock report, the HTML file looks like this:

![image](https://user-images.githubusercontent.com/6253590/71666779-b67e5900-2d62-11ea-97bb-4229e8c31dbc.png)

**TestSpec** has 0 feature and **AbstractSpec** does not appear inside the report itself.
It seems that in such a case, all features from the abstract class should appear as features from the implementation. Or that we should at least see the abstract class appear inside the report itself.

**Comment by renatoathaydes** [comment]
This bug has already been fixed and its fix will be in the next release!

**Comment by renatoathaydes** [comment]
Fixed in version 1.6.3.
